tests/worker/test_scoring.py

```diff
@@ -569,6 +569,171 @@ def test_score_package_graph(
         assert report.json_with_dependencies() == expected_report_json
 
 
+compare_package_graph_testcases = {
+    "same_one_node_graph": (
+        m.PackageGraph(
+            id=-1,
+            package_links_by_id={0: (0, 0)},
+            distinct_package_versions_by_id={
+                0: m.PackageVersion(id=0, name="test-solo-pkg", version="0.1.0"),
+            },
+            get_npmsio_scores_by_package_version_id=lambda: {
+                0: ("0.1.0", {"0.1.0": 0})
+            },
+            get_npm_registry_data_by_package_version_id=lambda: {0: None},
+            get_advisories_by_package_version_id=lambda: {0: []},
```

Comment:
can we remove `get_npmsio_scores_by_package_version_id`, `get_npm_registry_data_by_package_version_id`, and `get_advisories_by_package_version_id` from the test cases?